src/app/admin/admin-reports/filtered-collections/filtered-collections.component.html

```diff
@@ -0,0 +1,64 @@
+                          {{ "admin.reports.commons.filters" | translate }}
+                        </ng-template>
+                        <ng-template ngbPanelContent>
+                            <div class="container">
+                                <div class="row">
+                                    <span class="col-3"></span>
+                                    <button class="btn btn-primary mt-1 col-6" (click)="submit()">{{ "admin.reports.button.show-collections" | translate }}</button>
+                                </div>
+                                <ds-filters [filtersForm]="filtersFormGroup()"></ds-filters>
+                                <div class="row">
+                                    <span class="col-3"></span>
+                                    <button class="btn btn-primary mt-1 col-6" (click)="submit()">{{ "admin.reports.button.show-collections" | translate }}</button>
+                                </div>
+                            </div>
+                        </ng-template>
+                    </ngb-panel>
+                    <ngb-panel id="collections">
+                        <ng-template ngbPanelTitle>
+                          {{ "admin.reports.collections.collections-report" | translate }}
+                        </ng-template>
+                        <ng-template ngbPanelContent>
+                            <table id="table" class="table table-striped">
+                                <thead>
+                                    <tr class="header">
+                                        <th rowspan="2">{{ "admin.reports.collections.community" | translate }}</th>
+                                        <th rowspan="2">{{ "admin.reports.collections.collection" | translate }}</th>
+                                        <th>{{ "admin.reports.collections.nb_items" | translate }}</th>
+                                        <th>{{ "admin.reports.collections.match_all_selected_filters" | translate }}</th>
+                                        <th *ngFor="let filter of results.summary.values | keyvalue">{{ ("admin.reports.commons.filters." + getGroup(filter.key) + "." + filter.key) | translate }}</th>
+                                    </tr>
+                                    <tr class="header">
+                                        <th class="num">{{ results.summary.nbTotalItems }}</th>
+                                        <th class="num">{{ results.summary.allFiltersValue }}</th>
+                                        <th class="num" *ngFor="let filter of results.summary.values | keyvalue">{{ filter.value }}</th>
+                                    </tr>
+                                </thead>
+                                <tbody>
+                                    <tr *ngFor="let coll of results.collections">
+                                        <td><a href="{{ coll.communityHandle }}" target="_blank">{{ coll.communityLabel }}</a></td>
```

Comment:
All of these links are broken for me (return a 404).  It seems to be that they are missing the "/handle" path.  So, the link URL looks like this: `http://localhost:4000/123456789/498` when it should be `http://localhost:4000/handle/123456789/498`

NOTE: To also fix the CodeQL warning, you could add `rel="noopener noreferrer"` to this link.  I think the warning is a "false positive" as these are not truly external links, but it won't hurt to add the `rel`.

Reply:
Done. For uniformity, I added `rel="noopener noreferrer"` to the links in `filtered-items.component.html` too.